src/helperFunctions/object_conversion.py

```diff
@@ -14,7 +15,7 @@ def _add_firmware_only_fields(fo, meta):
 
 def _add_file_object_only_fields(fo, meta):
     if not isinstance(fo, Firmware):
-        meta['firmwares_including_this_file'] = list(fo.get_virtual_file_paths().keys())
+        meta['firmwares_including_this_file'] = list(fo.parent_firmware_uids)
```

Comment:
At some point we might want to change the db key as well to avoid confusion in the future